packages/odyssey-react/package.json

```diff
@@ -47,6 +49,7 @@
     "build:clean": "rm -rf dist",
```

Comment:
Since this is an open source repo, consider integrating [rimraf](https://github.com/isaacs/rimraf) at a later point in time for our Windows devs.